<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VMPDBG v0.1.0 (alpha)</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>main_ico.ico</normaloff>main_ico.ico</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Machine</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QSplitter" name="splitter_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QPlainTextEdit" name="plainTextEdit_ASM">
            <property name="font">
             <font>
              <family>Courier New</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QTreeWidget" name="treeWidget_REGISTERS">
            <column>
             <property name="text">
              <string notr="true">1</string>
             </property>
            </column>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Memory</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPlainTextEdit" name="plainTextEdit_MEMORY">
           <property name="font">
            <font>
             <family>Courier New</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Log</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPlainTextEdit" name="plainTextEdit_LOG"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>745</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Session"/>
    <addaction name="actionRestore_Session"/>
    <addaction name="actionSave_Session"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuDebugger">
    <property name="title">
     <string>Debugger</string>
    </property>
    <addaction name="actionStart_Debugg"/>
    <addaction name="actionStop_Debugg"/>
    <addaction name="actionBaby_Step"/>
    <addaction name="actionGiant_Step"/>
    <addaction name="actionRun"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <widget class="QMenu" name="menuBreakpoints">
     <property name="title">
      <string>Breakpoints</string>
     </property>
     <addaction name="separator"/>
     <addaction name="actionAdd_Breakpoint"/>
     <addaction name="actionRemove_Breakpoint"/>
    </widget>
    <addaction name="actionDump_Memory"/>
    <addaction name="menuBreakpoints"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDebugger"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_Session">
   <property name="text">
    <string>New Session</string>
   </property>
  </action>
  <action name="actionRestore_Session">
   <property name="text">
    <string>Restore Session</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/icons/Virus-50.png</normaloff>:/Icons/icons/Virus-50.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSave_Session">
   <property name="text">
    <string>Save Session</string>
   </property>
  </action>
  <action name="actionStart_Debugg">
   <property name="text">
    <string>Start Debug Session</string>
   </property>
  </action>
  <action name="actionStop_Debugg">
   <property name="text">
    <string>Stop Debug Session</string>
   </property>
  </action>
  <action name="actionBaby_Step">
   <property name="text">
    <string>Baby Step</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
  <action name="actionGiant_Step">
   <property name="text">
    <string>Giant Step</string>
   </property>
   <property name="shortcut">
    <string>F10</string>
   </property>
  </action>
  <action name="actionDump_Memory">
   <property name="text">
    <string>Dump Memory</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="text">
    <string>Run</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
  <action name="actionAdd_Breakpoint">
   <property name="text">
    <string>Add Breakpoint</string>
   </property>
  </action>
  <action name="actionRemove_Breakpoint">
   <property name="text">
    <string>Remove Breakpoint</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
